build/bom/pom.xml

```diff
@@ -286,6 +286,12 @@
             <classifier>loader</classifier>
          </dependency>
 
+         <dependency>
+             <groupId>org.infinispan</groupId>
+             <artifactId>testcontainers-infinispan</artifactId>
```

Comment:
All our artifacts start with prefix `infinispan-`. Should this be `infinispan-testcontainers`?